src/Microsoft.Health.Fhir.Tests.Common/TestFiles/R5/BloodGlucose.json

```diff
@@ -0,0 +1,72 @@
+﻿{
+  "resourceType": "Observation",
```

Comment:
We might be able to start a /TestFiles/Normative/ version folder for (R4+) tests on normative resources that don't change